runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/windmill/client/grpc/GrpcDispatcherClient.java

```diff
@@ -97,6 +97,14 @@ CloudWindmillServiceV1Alpha1Stub getWindmillServiceStub() {
         : randomlySelectNextStub(windmillServiceStubs));
   }
 
+  WindmillServiceAddress getWindmillServiceAddress() {
+    ImmutableList<HostAndPort> endpoints =
+        ImmutableList.copyOf(dispatcherStubs.get().dispatcherEndpoints());
```

Comment:
just use getDispatcherEndpoints?
it doesn't seem like we need to copy we just need to make sure we use a consistent set by keeping the reference to it.

Reply:
realized we don't need this since the cache handles all of it